datascroller/scroller.py

```diff
@@ -488,18 +494,18 @@ def key_press_and_print_df(stdscr, df):
 
         elif key == FILTER:
             query_bytes = get_user_input_with_prompt(stdscr, term_rows - 1, 0,
-                                                       "Column filter: ")
+                                                     "Column filter: ")
             query_string = query_bytes.decode(encoding="utf-8")
             if len(query_string) > 0:
                 try:
                     df_window = df_window.filter(query_string, viewing_area)
-                except pandasql.sqldf.PandaSQLException: # TODO better exception handling
```

Comment:
This was the actual error. You imported sqldf from pandassql. So you can't reference pandassql directly.